feature/bgp/policybase/ate_tests/route_installation_test/route_installation_test.go

```diff
@@ -561,7 +565,7 @@ func TestBGPPolicy(t *testing.T) {
 			bgp := bgpCreateNbr(dutAS, ateAS, tc.policy)
 			// Configure ATE to setup traffic.
 			allFlows := configureATE(t, ate)
-			dut.Config().NetworkInstance(*deviations.DefaultNetworkInstance).Protocol(telemetry.PolicyTypes_INSTALL_PROTOCOL_TYPE_BGP, "BGP").Bgp().Replace(t, bgp)
+			dut.Config().NetworkInstance(*deviations.DefaultNetworkInstance).Protocol(telemetry.PolicyTypes_INSTALL_PROTOCOL_TYPE_BGP, "BGP").Replace(t, bgp)
 			// Send and verify traffic.
```

Comment:
Will this change for specific vendor or it should work for all the vendors? The reason i am asking is i saw some vendor tests are passing with the current codes.

Reply:
I have certain limitations to test with various vendors, as to my knowledge, only 2 vendors provide open NOS: cEOS/SRL.
Other vendors should test those changes as well against their NOS.
Theoretically this *should* work since we're simply changing the config level from `NetworkInstance_Protocol_Bgp` to `NetworkInstance_Protocol`.